pkg/controllers/disruption/controller.go

```diff
@@ -221,31 +220,23 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command,
 	c.cluster.MarkForDeletion(providerIDs...)
 
 	if err := c.queue.Add(orchestration.NewCommand(nodeClaimNames,
-		lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode { return c.StateNode }), commandID, m.Type(), m.ConsolidationType())); err != nil {
+		lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode { return c.StateNode }), commandID, m.Reason(), m.ConsolidationType())); err != nil {
 		c.cluster.UnmarkForDeletion(providerIDs...)
 		return fmt.Errorf("adding command to queue (command-id: %s), %w", commandID, multierr.Append(err, state.RequireNoScheduleTaint(ctx, c.kubeClient, false, stateNodes...)))
 	}
 
 	// An action is only performed and pods/nodes are only disrupted after a successful add to the queue
-	DecisionsPerformedCounter.With(map[string]string{
-		actionLabel:            string(cmd.Action()),
-		methodLabel:            m.Type(),
+	DecisionsPerformedTotal.With(map[string]string{
+		decisionLabel:          string(cmd.Action()),
+		metrics.ReasonLabel:    m.Reason(),
 		consolidationTypeLabel: m.ConsolidationType(),
 	}).Inc()
-	for _, cd := range cmd.candidates {
-		PodsDisruptedCounter.With(map[string]string{
-			metrics.NodePoolLabel:  cd.nodePool.Name,
-			actionLabel:            string(cmd.Action()),
-			methodLabel:            m.Type(),
-			consolidationTypeLabel: m.ConsolidationType(),
-		}).Add(float64(len(cd.reschedulablePods)))
-	}
 	return nil
 }
 
 // createReplacementNodeClaims creates replacement NodeClaims
 func (c *Controller) createReplacementNodeClaims(ctx context.Context, m Method, cmd Command) ([]string, error) {
-	reason := fmt.Sprintf("%s/%s", m.Type(), cmd.Action())
+	reason := fmt.Sprintf("%s/%s", m.Reason(), cmd.Action())
 	nodeClaimNames, err := c.provisioner.CreateNodeClaims(ctx, cmd.replacements, provisioning.WithReason(reason))
```

Comment:
```suggestion
	nodeClaimNames, err := c.provisioner.CreateNodeClaims(ctx, cmd.replacements, provisioning.WithReason(m.Reason()))
```

This should just be reason, since cmd.Action() is just going to be `replace` in all cases where we are launching new nodeclaims